DQM/SiStripMonitorClient/test/test_SiStripDQM_OfflineTkMap.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+function die { echo $1: status $2; exit $2; }
+GT=`echo ${@} | python3 -c 'import Configuration.AlCa.autoCond as AC;print(AC.autoCond["run3_data_prompt"])'`
+RUN="319176"
+REMOTE="/eos/cms/store/group/comm_dqm/DQMGUI_data/Run2018/ZeroBias/R0003191xx/DQM_V0001_R000319176__ZeroBias__Run2018B-PromptReco-v2__DQMIO.root"
```

Comment:
@smuzaffar it appears that PR integration tests are unable to see this file (see [unit test log](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-cf236d/17937/unitTests/src/DQM/SiStripMonitorClient/test/testSiStripDQM_OfflineTkMap/testing.log)).
What's the policy to make such file visible from tests? Should it be copied somewhere else?
Thanks.

Reply:
@mmusich , eos is not mounted on build nodes. You should use LFN e.g. `/store/group/comm_dqm/DQMGUI_data/Run2018/ZeroBias/R0003191xx/DQM_V0001_R000319176__ZeroBias__Run2018B-PromptReco-v2__DQMIO.root` and let cmsRun open the file using xrootd

Reply:
@smuzaffar I am afraid that what you suggest doesn't work here. The input root file is not used as event source in the `cmsRun` process, but it is just opened to access the DQM histograms in ROOT format. 
With this patch:
```diff
diff --git a/DQM/SiStripMonitorClient/test/test_SiStripDQM_OfflineTkMap.sh b/DQM/SiStripMonitorClient/test/test_SiStripDQM_OfflineTkMap.sh
index 94f662a60d5..5e6797944cb 100755
--- a/DQM/SiStripMonitorClient/test/test_SiStripDQM_OfflineTkMap.sh
+++ b/DQM/SiStripMonitorClient/test/test_SiStripDQM_OfflineTkMap.sh
@@ -2,15 +2,6 @@
 function die { echo $1: status $2; exit $2; }
 GT=`echo ${@} | python3 -c 'import Configuration.AlCa.autoCond as AC;print(AC.autoCond["run3_data_prompt"])'`
 RUN="319176"
-REMOTE="/eos/cms/store/group/comm_dqm/DQMGUI_data/Run2018/ZeroBias/R0003191xx/DQM_V0001_R000319176__ZeroBias__Run2018B-PromptReco-v2__DQMIO.root"
-DQMFILE="DQM_V0001_R000319176__ZeroBias__Run2018B-PromptReco-v2__DQMIO.root"
-if [ -f "${DQMFILE}" ]; then
-    rm $DQMFILE
-fi
-# copy file from eos
-xrdcp $REMOTE $DQMFILE
-if [ ! -f "${DQMFILE}" ]; then
-  die "SKIPPING test, file ${DQMFILE} not found" 0
-fi
+DQMFILE="/store/group/comm_dqm/DQMGUI_data/Run2018/ZeroBias/R0003191xx/DQM_V0001_R000319176__ZeroBias__Run2018B-PromptReco-v2__DQMIO.root"
 echo "Using file ${DQMFILE} and run ${RUN}. Running in ${LOCAL_TEST_DIR}."
 (cmsRun "${LOCAL_TEST_DIR}/SiStripDQM_OfflineTkMap_Template_cfg_DB.py" globalTag="$GT" runNumber="$RUN" dqmFile="$DQMFILE" detIdInfoFile="file.root") || die 'failed running SiStripDQM_OfflineTkMap_Template_cfg_DB.py' $?
```

the test fails in a local checkout.

Reply:
try using 
```
root://eoscms.cern.ch//eos/cms/store/group/comm_dqm/DQMGUI_data/Run2018/ZeroBias/R0003191xx/DQM_V0001_R000319176__ZeroBias__Run2018B-PromptReco-v2__DQMIO.root
```
at least I can copy/access it using 
```
edmCopyUtil root://eoscms.cern.ch//eos/cms/store/group/comm_dqm/DQMGUI_data/Run2018/ZeroBias/R0003191xx/DQM_V0001_R000319176__ZeroBias__Run2018B-PromptReco-v2__DQMIO.root .
```
of course this way our system is not going to cache this file and when this file is deleted then the unit test will start failing

Reply:
using the `xrootd` global redirector seems to work. I'll push a new commit soon.